src/vs/base/common/buffer.ts

```diff
@@ -14,8 +14,8 @@ let textDecoder: TextDecoder | null;
 
 export class VSBuffer {
 
-	static alloc(byteLength: number): VSBuffer {
-		if (hasBuffer) {
+	static alloc(byteLength: number, forceUseUint8Array = false): VSBuffer {
```

Comment:
Exposing this low level flag feels wrong, but as I noted in the issue comment I'm not sure if there is a better way to do this